src/components/VisuItem/VisuItem.styles.tsx

```diff
@@ -19,6 +19,29 @@ export const ItemWrapper = styled.div`
   border-radius: 4px;
 `;
 
+const spinnerAnimation = keyframes`
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+`;
```

Comment:
```suggestion
```

We actually can reuse the "spin" keyframe animation from the `globalStyles.tsx` file